koswat/dike_reinforcements/input_profile/reinforcement_input_profile_calculation_protocol.py

```diff
@@ -16,6 +17,7 @@ class ReinforcementInputProfileCalculationProtocol(BuilderProtocol, Protocol):
     base_profile: KoswatProfileProtocol
     reinforcement_settings: KoswatReinforcementSettings
     scenario: KoswatScenario
+    reinforced_data: KoswatInputProfileBase
```

Comment:
you should not need this

Reply:
Yes, having a look at the code in a local checkout I can also say that it does not make sense to have it either at `ReinforcementInputProfileCalculationBase` so it should be safe to remove it from there as well.

Reply:
With the new approach of creating the reinforcement profile it is not needed anymore, indeed.